api/bag_vrnodedescriptor.cpp

```diff
@@ -55,7 +58,7 @@ std::shared_ptr<VRNodeDescriptor> VRNodeDescriptor::create(
     int compressionLevel)
 {
     return std::shared_ptr<VRNodeDescriptor>(
-        new VRNodeDescriptor{dataset.getNextId(), chunkSize,
+        new VRNodeDescriptor{dataset.getNextId(), 1, 1, chunkSize,

```

Comment:
@brian-r-calder I changed this to 1 row and 1 column to get the existing tests to pass, but it seems like this should start out as 0, 0 and only be changed if a node is added. What do you think?